docs/specification/May2019Ballot.md

```diff
@@ -353,6 +353,8 @@ FHIR is the interoperable data model used by FHIRcast. The fields within `contex
 
 A FHIR server may not support the *\_elements* query parameter; a subscriber MUST gracefully handle receiving a full FHIR resource in the context of a notification.
 
+The name of the event SHOULD succinctly and clearly describe the activity or event. Event names are unique so event creators SHOULD take care to ensure newly proposed hooks do not conflict with an existing event name. Event creators SHALL name their event with reverse domain notation (e.g. `org.example.patient-transmogrify`) if the event is specific to an organization. Reverse domain notation SHALL not be used by a standard event catalog.
```

Comment:
> ...newly proposed hooks do not...

We use _hooks_ here to refer to events, which we don't do anywhere else in the spec from what I can tell. Or is _hooks_ more or less equivalent with _events_ here in this context you think? I can see that it might be.

Reply:
Ahh, great catch Leo. Fixed.